src/ServiceControl.Monitoring.Data/RawDataReporter.cs

```diff
@@ -77,7 +78,7 @@ public void Start()
                         }
 
                         totalSpinningTime += singleSpinningTime;
-                        await Task.Delay(singleSpinningTime).ConfigureAwait(false);
+                        await Task.Delay(singleSpinningTime, cancelReportingTokenSource.Token).ConfigureAwait(false);
```

Comment:
Need to catch OperationCancelledException, probably surrounding the whole async Func body.

Reply:
Discussed and we decided to not try catch